### PR TITLE
Also run Kubernetes end to end tests under Kubernetes v1.24.0

### PR DESCRIPTION
**Pull request by Kami**
Small change to also run Kubernetes end to end tests under latest Kubernetes version - v1.24.0.

### CODE REVIEW

**Comment by Kami** [comment]
Right now blocked by minikube bug - https://github.com/kubernetes/minikube/issues/13945.

**Comment by Kami** [comment]
minikube v1.26.0 with the bug fix has been released and the build is now passing.

File: .github/actions/setup-minikube-cluster/action.yml [hunk 1]
```diff
@@ -31,7 +31,7 @@ runs:
       # See https://github.com/actions/runner/issues/834
       uses: ChristopherHX/conditional@b4a9649204f81002ec9a4ef7d4bf7d6b2ab7fa55
       with:
-        if: ${{ inputs.k8s_version != '' }}
+        if: ${{ inputs.k8s_version != '' && inputs.k8s_version != 'v1.24.0' }}
```
Comment (dlobue) [inline .github/actions/setup-minikube-cluster/action.yml:34]: Why disallow this k8s version?

File: .github/actions/setup-minikube-cluster/action.yml [hunk 2]
```diff
@@ -41,6 +41,22 @@ runs:
             driver: '${{ inputs.minikube_driver }}'
             start args: '--container-runtime=${{ inputs.container_runtime }}'
 
+    - name: Create minikube Kubernetes ${{ inputs.k8s_version }} Cluster
+      id: create-minikube-cluster-124
+      # TODO: Workaround until conditions are supported natively
+      # See https://github.com/actions/runner/issues/834
```
Comment (dlobue) [inline .github/actions/setup-minikube-cluster/action.yml:47]: That issue has been resolved, and conditions are supported natively on steps:
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsif
https://github.blog/changelog/2021-11-09-github-actions-conditional-execution-of-steps-in-actions/
https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#runs-for-composite-actions

**Review by dlobue — CHANGES_REQUESTED** [review]

File: .github/actions/setup-minikube-cluster/action.yml (hunk 1 above)
Comment (dlobue) [reply .github/actions/setup-minikube-cluster/action.yml:34]: nvm. I see your comment: https://github.com/scalyr/scalyr-agent-2/pull/885#issuecomment-1132657663

Comment (Kami) [reply .github/actions/setup-minikube-cluster/action.yml:34]: We don't set "start args" action parameter for v1.24.0 (see step below).

**Pull request merged**